sdk/lib/atl/atlimage.h

```diff
@@ -130,9 +142,9 @@ class CImage
         bf.SourceConstantAlpha = bSrcAlpha;
         bf.AlphaFormat = AC_SRC_ALPHA;
 
-        GetDC();
+        HDC hSrcDC = GetDC();
```

Comment:
Why?
The GetDC() function seems to reset the m_hDC, so that here you would basically get hSrcDC to be == m_hDC.
So I'm not sure your code change here actually changes the behaviour.

Also I wonder whether the AlphaBlend method should be annotated with a `throw()`.

Reply:
`GetDC()` usage is fixed.

`AlphaBlend` has `throw()`, according to Doxygen analysis:

![image](https://github.com/reactos/reactos/assets/2107452/cad1fc7f-ad05-4641-81ea-ea1279e8804b)

Reply:
They will have to be replaced with `noexcept`.